examples/proxy/record-encryption/PREPARE_KMS.md

```diff
@@ -127,3 +129,67 @@ Finally, delete the .token files:
 ```sh
 rm *.token
 ```
+
+# Preparing AWS KMS
+
+To use [AWS KMS](https://docs.aws.amazon.com/kms/latest/developerguide/overview.html) for record encryption, follow these steps. 
+
+### Prerequisites
+
+* An AWS account with the following privileges:
+  * The ability to create keys in AWS KMS.
+  * The ability to create an application identity for use by the filter. This includes creating an IAM user with an access key, and creating and attaching policies.
+* The following tools are installed:
+  * [AWS CLI ](https://aws.amazon.com/cli/).
+  * GNU `sed`
+  * `jq`
+
+
+Note: these examples are prepared against AWS CLI 2.13.17.
+
+### Configuring an Application Identity
+
+Define a variable containing your AWS account id.
+
+```sh
+AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+```
+
+You need to create an AWS IAM for use by Record Encryption:
+
+```sh
+aws iam create-user --user-name kroxylicious
```

Comment:
Should we use `kroxylicious` I think thats the first reference to the upstream project in the docs.

Though I admit its a better username than `streamsForApacheKafkaProxyRecordEncryptionFilter`

Reply:
proxyUser?

Reply:
Probably `kafkaProxyUser` for consistency's sake